src/tests/test_alldebrid_downloader.py

```
@@ -46,91 +51,51 @@ def get(url, **params):
-def test_get_torrents(status_all):
-    alldebrid.get = status_all
-    torrents = get_torrents()
-    assert torrents[0]["status"] == "Ready"
-
-
-def test_delete(delete):
-    alldebrid.get = delete
-    delete(123)
+def test_delete_torrent(downloader):
+    assert (
+        downloader.delete_torrent(MAGNET_ID) == None
```

Ruff (0.8.0)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)